frontend/src/routes/+page.svelte

```diff
@@ -1,109 +1,137 @@
-			queryClient.invalidateQueries('get-todos');
-		}
-	});
-
-	const updateTodoMutation = useMutation(updateTodo, {
-		onSuccess: () => {
-			queryClient.invalidateQueries('get-todos');
-		}
-	});
-
-	const deleteTodoMutation = useMutation(deleteTodo, {
-		onSuccess: () => {
-			queryClient.invalidateQueries('get-todos');
-		}
-	});
-
-	import { onMount } from 'svelte';
-
-	onMount(async () => {});
-
-	const handleCreateButtonClick = async () => {
-		selectedTodo = {
-			Id: 0,
-			Title: '',
-			Description: '',
-			CompletedAt: undefined
-		} as Todo;
-	};
-
-	$: {
-		console.log('parend', selectedTodo);
-	}
 </script>
 
-<div class="flex pt-5 justify-center w-full h-full bg-zinc-800 text-zinc-200 font-sans px-5">
-	<div class="max-w-4xl w-full h-full flex flex-col">
-		<div class="flex gap-2">
-			<div class="flex-1">
-				<SearchBar on:search={handleSearchEvent} />
-			</div>
-
-			<button
-				class="h-full px-4 bg-green-500 text-base rounded-[4px]"
-				on:click={handleCreateButtonClick}
-			>
-				Create New</button
-			>
-		</div>
-
-		<div class="flex mt-8 gap-5 flex-1 min-h-0">
-			<div class="flex-[0.5] overflow-y-scroll scroll-bar">
-				{#if $todoQueryResult.data}
-					<TodoBar todos={$todoQueryResult.data} on:todo-selected={handleTodoSelected}></TodoBar>
-				{/if}
+<main style="background-color: #bae8cb">
+	<NavBar />
+	<br>
+	<div>
+		<h3 class="h3 font-bold mb-5 mt-16 p-15 text-gray-950 align-middle origin-center text-center">Welcome to</h3>
+		<h1 class="h1 font-bold mb-10 mt-5 p-15 text-gray-950 align-middle origin-center text-center">QuickTable</h1>
+	</div>
+	<div class="p-5 flex justify-center h-screen">
+		<form class="flex flex-col items-center">
+			<label for="searchName" class="text-gray-950 align-middle origin-center text-center mb-1">Search by Name</label>
+			<div class="flex">
+				<input type="text" name="searchName" id="searchNameHome" placeholder="Search..." class="w-64 py-2 px-4 rounded-full border border-gray-300 bg-white focus:outline-none focus:ring focus:border-blue-300 align-middle origin-center relative text-gray-950">
+				<button type="submit" value="searchName" class="py-1 px-2 rounded-full border border-gray-300 bg-white focus:outline-none focus:ring focus:border-blue-300 text-gray-950">
+					<img src="img/mag.svg" alt="Submit" style="width: 20px; height: 20px;">
+				</button>
 			</div>
-
-			<div class="flex-[0.5] py-2">
-				<ManageTodo
-					todo={selectedTodo}
-					on:create-todo={(event) => {
-						$createTodoMutation.mutate(event.detail);
-					}}
-					on:delete-todo={(event) => {
-						$deleteTodoMutation.mutate(event.detail.Id);
-					}}
-					on:update-todo={(event) => {
-						$updateTodoMutation.mutate(event.detail);
-					}}
-				/>
+			<h5 class="h5 font-bold text-gray-950 align-middle origin-center text-center p-5">or</h5>
+			<label for="searchTime" class="text-gray-950 mb-1 align-middle origin-center text-center">Search by Time</label>
+			<div class="flex justify-center">
```

Comment:
Is this supossed to emulate a date input? Could you use a native one like  https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date or one provided by skeleton ? https://www.skeleton.dev/elements/forms

Reply:
Wow, this seems way easier. Thank you for showing me this!